src/python/espressomd/observables.py

```diff
@@ -14,15 +14,19 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import numpy as np
 from .script_interface import ScriptInterfaceHelper, script_interface_register
 
 
 @script_interface_register
 class Observable(ScriptInterfaceHelper):
     _so_name = "Observables::Observable"
-    _so_bind_methods = ("calculate", "n_values")
+    _so_bind_methods = ("_calculate", "shape")
```

Comment:
why do you rename it? We should stick to our naming conventions.

Reply:
just tried it, but
```python
def calculate(self):
    return np.array(self.call_method("calculate")).reshape(self.shape())
```
is overshadowed by the `self.calculate()` method created by the `ScriptInterfaceHelper`.